tests/test_kex.c

```diff
@@ -23,12 +23,6 @@ struct kex_testcase {
 
 /* Add new testcases here */
```

Comment:
We should remove test_kex.c

Reply:
I'll remove it in issue: https://github.com/open-quantum-safe/liboqs/issues/372